libraries/hardwareInterfaces.js

```diff
@@ -344,14 +363,14 @@ exports.clearTool = function(objectName, frameName){
             }
         }
     }
-}
+};
 
-exports.setTool = function(objectName, frameName, tool, dirName){
-  
-    var objectID = utilities.getObjectIdFromTarget(objectName, objectsPath);
-    console.log('set new tool: ', tool);
+exports.setTool = function (object, tool, newTool, dirName) {
```

Comment:
the naming of tool and newTool here suggests that they're of the same data type/format but the code below appears to use them in different ways

Reply:
this doesn't matter in the grand scheme of things